packages/dumb/src/components/stats/common/cbg-utils.tsx

```diff
@@ -50,10 +48,10 @@ export const computeCBGStyle = (value: number, bgClasses: BgClasses): CBGStyle =
     return { backgroundColor: styles['low-background'], color: styles['low-color'], left: '0' }
   }
   if (value > highValue) {
-    return { color: styles['high-color'], backgroundColor: styles['high-background'], left: `${CBG_BAR_WIDTH_IN_PX}px` }
+    return { color: styles['high-color'], backgroundColor: styles['high-background'], left: '100%' }
   }
   const cbgBarRange = highValue - veryLowValue // Number of value included in the cbg bar range (default is from 54 to 250)
-  const left = `${Math.round(((value - veryLowValue) * CBG_BAR_WIDTH_IN_PX) / cbgBarRange)}px`
+  const left = `${Math.round(((value - veryLowValue) * 100) / cbgBarRange)}%`
```

Comment:
I suggest keeping a const to store the value `100` because it is unclear what it is (magic number @celia-rouis ) :teacher: